public/js/components/Breakpoints.js

```diff
@@ -65,6 +67,10 @@ const Breakpoints = React.createClass({
     this.props.selectSource(sourceId, { line });
   },
 
+  removeBreakpoint(breakpoint) {
+    this.props.removeBreakpoint(breakpoint.location);
```

Comment:
You need to receive the event here and do `event.stopPropagation()` because it's still highlighting the breakpoint as if you click on the breakpoint item. Stopping the propagation will block the item from receiving the click event.